src/main/java/org/intellij/lang/batch/runner/BatchConfigurationType.java

```diff
@@ -15,6 +14,21 @@
  * @author wibotwi
  */
 public class BatchConfigurationType implements ConfigurationType {
+    public static BatchConfigurationType getInstance() {
+        ConfigurationType[] configurationTypes = CONFIGURATION_TYPE_EP.getExtensions();
+
+        for (ConfigurationType configurationType : configurationTypes) {
+            if (configurationType instanceof BatchConfigurationType) {
+                return (BatchConfigurationType) configurationType;
+            }
+        }
+
+        assert false;

```

Comment:
Some magic?

Reply:
I ran the autoformatter of IntelliJ, which moved this method from line 35 to 17 (see the diff)

Reply:
Oh, ok. Can you please fix conflics here?